.integration/tests/dpc_testnet1.rs

```diff
@@ -419,7 +418,7 @@ fn test_testnet1_dpc_execute_constraints() {
 
     // NOTE: Do not change this to `Testnet1Parameters::inner_circuit_id()` as that will load the *saved* inner circuit VK.
     let inner_circuit_id = <Testnet1Parameters as Parameters>::inner_circuit_id_crh()
-        .hash_field_elements(&inner_snark_vk.to_field_elements().unwrap())
+        .hash_bits(&inner_snark_vk.to_minimal_bit_representation())
```

Comment:
We now use `hash_bits` to hash a bit array, and use `to_minimal_bit_representation` to convert a struct to its minimal array representation.